ios/Sources/AppFeature/AppFeature.swift

```diff
@@ -0,0 +1,36 @@
+import ComposableArchitecture
+import HomeFeature
+
+public struct AppState: Equatable {
+    public var homeState: HomeState
```

Comment:
Because of this [reason](https://github.com/DroidKaigi/conference-app-2021/pull/457#discussion_r644103943), leave this property to public too. 🙏🏼

Reply:
How about using `private(set)`?

Reply:
> How about using private(set)?

It's a good idea!
Is it means using `private(set)` accessor to this property only? If so, I agree!
I think it's good because the accessor can explicitly show that it will not be changed from outside.

I think the properties of other states are not necessarily the same.

Reply:
I think it would be nice to apply to this and https://github.com/DroidKaigi/conference-app-2021/pull/457#discussion_r644103943.

Reply:
I don't think so too 😢 

The reason why I thought that is other modifiable properties must allow at least internal changes from the outside to allow Reducer to change properties.

Reply:
I'm sorry, this line should be changed from outside because in `appReducer`, pullbacks `homeReducer` and pass `homeState` with writable property is needed.

Reply:
👌